scripts/seed.ts

```diff
@@ -111,6 +129,39 @@ async function main() {
     }
   }
 
+  console.info("Seeding creator requests...");
+  const seedCreatorRequests: SeedCreatorRequests[] = [
+    {
+      userId: createdUserIds[0],
+      portfolioLink: "https://photos.example.com/alice",
+      status: "pending",
+      submittedAt: new Date(),
+      motivation: "I just love sharing photos",
+    },
+    {
+      userId: createdUserIds[2],
+      portfolioLink: "https://runcam.dev/charlie",
+      status: "approved",
+      note: "Portfolio verified",
+      submittedAt: new Date(),
+      motivation: "Covering community races weekly",
+    },
+    {
+      userId: createdUserIds[3],
+      portfolioLink: "https://photos.example.com/diana",
+      status: "approved",
+      note: "Portfolio verified",
+      submittedAt: new Date(),
+      motivation: "Share Bali triathlons",
+    },
```

Comment:
Add the `reviewed_by` section on this. Make both reviewed by Eve.
```suggestion
    {
      userId: createdUserIds[2],
      portfolioLink: "https://runcam.dev/charlie",
      status: "approved",
      note: "Portfolio verified",
      reviewedBy: createdUserIds[4],
      submittedAt: new Date(),
      motivation: "Covering community races weekly",
    },
    {
      userId: createdUserIds[3],
      portfolioLink: "https://photos.example.com/diana",
      status: "approved",
      note: "Portfolio verified",
      reviewedBy: createdUserIds[4],
      submittedAt: new Date(),
      motivation: "Share Bali triathlons",
    },
```

Also sync the `SeedCreatorRequests` type above.